ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -4,16 +4,26 @@
 import { generateKeyFromPrivateKey, wordsToSpendingKey } from '@ironfish/rust-nodejs'
 import {
   ACCOUNT_SCHEMA_VERSION,
-  AccountImport,
   Bech32m,
   JSONUtils,
   PromiseUtils,
+  RpcResponseEnded,
 } from '@ironfish/sdk'
 import { CliUx, Flags } from '@oclif/core'
 import { IronfishCommand } from '../../command'
 import { RemoteFlags } from '../../flags'
 import { LANGUAGE_VALUES } from '../../utils/language'
 
+type AccountImport = {
+  name: string
+  spendingKey?: string
+  viewKey?: string
+  publicAddress?: string
+  incomingViewKey?: string
+  outgoingViewKey?: string
+  version: number
+}
```

Comment:
Split into two types, this will make code below much cleaner.

Reply:
I ran into a roadblock doing that, but didn't document what it was, hm. Will retry/re-do and see where we get.

Reply:
Hm, worked fine! Wonder what I was doing wrong last time.